database-ideas/load_shot_data.py

```diff
@@ -0,0 +1,646 @@
+########################### DO NOT MODIFY THIS SECTION ##########################
+#################################################################################
+import sqlite3
+from sqlite3 import Error
+import csv
+
+import json
+from pathlib import Path
+import pprint
+
+import requests
+#################################################################################
+
+## Change to False to disable Sample
+SHOW = True
+
+############### SAMPLE CLASS AND SQL QUERY ###########################
+######################################################################
+class Sample():
+    def sample(self):
+        try:
+            connection = sqlite3.connect("sample")
+            connection.text_factory = str
+        except Error as e:
+            print("Error occurred: " + str(e))
+        print('\033[32m' + "Sample: " + '\033[m')
+        
+        # Sample Drop table
+        connection.execute("DROP TABLE IF EXISTS sample;")
+        # Sample Create
+        connection.execute("CREATE TABLE sample(id integer, name text);")
+        # Sample Insert
+        connection.execute("INSERT INTO sample VALUES (?,?)",("1","test_name"))
+        connection.commit()
+        # Sample Select
+        cursor = connection.execute("SELECT * FROM sample;")
+        print(cursor.fetchall())
+
+######################################################################
+
+# I want to test this on a single game first
+TEST_GAME_ID = 20001.0
+TEST_GAME_SEASON = 2023.0
```

Comment:
Maybe we can convert this into integers.

Reply:
It would look liek this.
```suggestion
TEST_GAME_ID = 20001
TEST_GAME_SEASON = 2023
```